remote-api/src/main/resources/alfresco/public-rest-context.xml

```diff
@@ -888,6 +888,8 @@
         <property name="nodes" ref="Nodes" />
         <property name="validator" ref="nodeValidator"/>
         <property name="ruleService" ref="RuleService" />
+        <property name="nodeService" ref="NodeService"/>
+        <property name="runtimeRuleService" ref="ruleService"/>
```

Comment:
Is it deliberate to use the internal (lowercase) `ruleService` rather than the `RuleService`?